python/ForceSMIP_trend_skill.py

```diff
@@ -0,0 +1,234 @@
+# %% imports
+import xcdat as xc
+import glob
+import numpy as np
+import xarray as xr
+from fx import get_delta_trend, seasonal_average, global_mean, plot_taylor
+
+# %% Parameters
+variable = 'tas'
+dpath = '/global/cfs/cdirs/m4581/pochedls/ForceSMIP_Tier1_final/'
+members = ['1B','1D','1E','1G','1J']
+season = 'annual' # annual, DJF, MAM, JJA, SON
+obs_member = '1I'
+startyear = 1980
+endyear = 2022
+
+# loop up dictionaries
+variableDict = {'monmaxtasmax': 'tasmax',
+                'monmaxpr': 'pr',
+                'monmintasmin': 'tasmin',
+                'zmta': 'ta'}
+plot_params = {'tos': {'xmin': 0.6, 'xmax': 1.3, 'ymax': 0.7, 'cmax': 0.8, 'cticks': 25},
+               'tas': {'xmin': 0.6, 'xmax': 1.2, 'ymax': 0.55, 'cmax': 0.7, 'cticks': 22},
+               'pr': {'xmin': 0., 'xmax': 1.3, 'ymax': 2.15, 'cmax': 2.4, 'cticks': 25},
+               'psl': {'xmin': 0., 'xmax': 1.3, 'ymax': 1.4, 'cmax': 1.8, 'cticks': 19},
+               'monmaxpr': {'xmin': 0., 'xmax': 1.3, 'ymax': 3.32, 'cmax': 3.6, 'cticks': 19},
+               'monmaxtasmax': {'xmin': 0.6, 'xmax': 1.2, 'ymax': 0.57, 'cmax': 0.7, 'cticks': 22},
+               'monmintasmin': {'xmin': 0.4, 'xmax': 1.3, 'ymax': 0.8, 'cmax': 1., 'cticks': 21},
+               'zmta': {'xmin': 0.9, 'xmax': 1.2, 'ymax': 0.4, 'cmax': 0.45, 'cticks': 19}}
+
+# %% get files
+submission_files = glob.glob(dpath + 'submissions_standardized/*nc')
+emean_files = glob.glob(dpath + 'ensmeans/*nc')
+ref_files = glob.glob(dpath + 'Evaluation-Tier1/*nc')
+
+# get variable id
+if variable in variableDict.keys():
+    varnam = variableDict[variable]
+else:
+    varnam = variable
+
+# %% loop over members of interest
+for j, member in enumerate(members):
+    print(str(j+1) + ' / ' + str(len(members)) + ': ' + member)
+    # get simplicity order
+    if variable == 'zmta':
+        simplicity_order = [np.nan, 2, 1, 12, 13, 17, 18, 14, 10, 15, 16, 4, np.nan, 8, 9, np.nan, 19, 20, 3, np.nan, np.nan, 5, 11, np.nan, 22, 21, np.nan, 6, 7, np.nan]
+    else:
+        simplicity_order = [27, 3, 1, 16, 17, 21, 22, 18, 14, 19, 20, 7, 8, 12, 13, 26, 23, 28, 5, 24, 2, 9, 15, 6, 30, 29, 25, 10, 11, 4]
+    # get submission, ensemble mean, and reference files
+    submission_file = [fn for fn in submission_files if '/' + variable + '_' + member in fn][0]
+    ## NOTE: Why do we take 1A if it is obs?
+    ## I assume this lets the rest of the code run and you just ignore some
+    ## results for observations?
```

Comment:
This was something I couldn't figure out when I was converting the code to Python, but I don't think the conditional below (`member == obs_member`) is ever met in the current code.